server/models/review_model.py

```diff
@@ -88,3 +88,12 @@ def delete_all(cls):
         except:
             print("Unexpected error:", sys.exc_info()[0])
             return {'error': 'Something went wrong'}, 500
+
+    @classmethod
+    def get_reviews_from_reviewee_id(cls, reviewee_id):
+        try:
+            reviews = cls.query.filter(reviewee_id=reviewee_id).all()
+            return { 'reviews' : reviews}
+        except:
+            return { 'error': "something went wrong :( "}
+            
```

Comment:
theres already a method to return reviews from reviewee_id. Its the `get_reviews(cls, id, requester) ` one. Any particular reason in using a new one?